### ISSUE TITLE
[Noto Color Emoji] Choice to include in lean4monaco

### ISSUE THREAD

**Issue by desaxce** [hhu-adam/lean4monaco#2]
Thank your for the constant improvements! 

I have a pipeline which leverages `lean4monaco` and automatically uses the latest published NPM package. 
During the week, I noticed a 10s degradation in load times for the Lean4 editor (I was in the countryside with a poor connection, but it was fast earlier last week).

It seems the introduction of the Noto Color Emoji font in [e1d06209](https://github.com/hhu-adam/lean4monaco/commit/e1d06209daf9f870e103ad12a75b08d560b569df) accounts for 24MB:
![network](https://github.com/user-attachments/assets/869857a5-a912-44d2-8002-11f121edc722)

VSCode probably comes bundled with those fonts, but web browsers download them. 

Could we bring down the load time by switching to a lighter font / working without it?

**Comment by joneugster** [comment]
Absolutely, I included the emoji font as Lean has a few emojis being used, which look bad in JuliaMono's black-and-wwhite. Most concretely its about ✅ and ❌.

If you have a suggestion for a lighter emoji font, I think we can just switch them out.

**Comment by joneugster** [comment]
Absolutely, I included the emoji font as Lean has a few emojis being used, which look bad in JuliaMono's black-and-wwhite. Most concretely its about ✅, ❌, (and 💥).



If you have a suggestion for a lighter emoji font, I think we can just switch them out.

**Comment by joneugster** [comment]
I've tried to create my own font with just these emojis, but apparentoy its not compatible with some devices/browsers. That one is here:

https://github.com/hhu-adam/lean4monaco/tree/main/src/fonts/LeanWeb

**Comment by desaxce** [comment]
Thanks @joneugster, I got back to it because I faced the slow initial load once again. 

I opened https://github.com/hhu-adam/lean4monaco/pull/4 to replace Noto with the one you created.

**Comment by desaxce** [comment]
Closing, see comments on https://github.com/hhu-adam/lean4monaco/pull/4

**Comment by joneugster** [comment]
I've created a new Issue #12, contributions welcome!
